metron-deployment/packaging/ambari/README.md

```diff
@@ -0,0 +1,85 @@
+# Ambari Management Pack Development
+Typically, Ambari Management Pack development will be done in the Vagrant environments. These instructions are specific to Vagrant, but can be adapted for other environemnts (e.g. make sure to be on the correct nodes for server vs agent files)
+
+
+# Adding a new property
+1. Add the property to the appropriate `*-env.xml` file found in `METRON.CURRENT/configuration`.
+  ```
+    <property>
+        <name>new_property</name>
+        <description>New Property description</description>
+        <value>Default Value</value>
+        <display-name>New Property Pretty Name</display-name>
+    </property>
+  ```
+
+2.  Reference the property in `METRON.CURRENT/package/scriptes/params/params_linux.py`, unless it will be used in Ambari's status command.  It will be stored in a variable. The name doesn't have to match, but it's preferred that it does.
+  ```
+  new_property = config['configurations']['metron-env']['new_property']
```

Comment:
We have specific *-env.xml files for different components.  Should we point that out here like we do in the previous step?  I'm not sure people will know to change ['metron-env'] to the right env file.